src/nbl/asset/interchange/CPLYMeshWriter.cpp

```diff
@@ -78,44 +78,81 @@ bool CPLYMeshWriter::writeAsset(io::IWriteFile* _file, const SAssetWriteParams&
     io::IWriteFile* file = _override->getOutputFile(_file, ctx, {mesh, 0u});
 
     auto meshbuffers = mesh->getMeshBuffers();
-	if (!file || !mesh || meshbuffers.size() > 1)
+	if (!file || !mesh)
 		return false;
+    
+    if (meshbuffers.size() > 1)
+    {
+        #ifdef  _NBL_DEBUG
+        os::Printer::log("PLY WRITER ERROR (" + std::to_string(__LINE__) + " line): Only one meshbuffer input is allowed for writing!", ELL_ERROR);
+        #endif // _NBL_DEBUG
+        return false;
```

Comment:
dont fail, just only write the first one

Reply:
corrected in following commit